clients/java/src/main/java/marquez/client/models/RawLineageEvent.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018-2022 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.client.models;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+import lombok.Value;
+
+@Value
+public class RawLineageEvent {
+  String eventType;
+  ZonedDateTime eventTime;
+  Map<String, Object> run;
+  Map<String, Object> job;
+  List<Object> inputs;
+  List<Object> outputs;
+  String producer;
```

Comment:
Use the `URI` type since `producer` must be a valid URI?

Reply:
Done.